decidim-initiatives/app/cells/decidim/initiatives/content_blocks/highlighted_initiatives_settings_form_cell.rb

```diff
@@ -10,9 +10,20 @@ def content_block
           options[:content_block]
         end
 
-        def label
+        def max_results_label
           I18n.t("decidim.initiatives.admin.content_blocks.highlighted_initiatives.max_results")
         end
+
+        def order_label
+          I18n.t("decidim.initiatives.admin.content_blocks.highlighted_initiatives.order.label")
+        end
+
+        def order_select
+          [
+            [I18n.t("decidim.initiatives.admin.content_blocks.highlighted_initiatives.order.default"), 0],
+            [I18n.t("decidim.initiatives.admin.content_blocks.highlighted_initiatives.order.most_recent"), 1]
```

Comment:
We're saving this setting as a boolean value, but this won't escale if we try to add more sorting orders. Can you move this from boolean to strings? Something like `"default"` and `"most_recent"` would work